pkg/reconciler/taskrun/controller.go

```diff
@@ -40,18 +40,8 @@ import (
 	"knative.dev/pkg/logging"
 )
 
-// ControllerConfiguration holds fields used to configure the
-// TaskRun controller.
-type ControllerConfiguration struct {
-	// Images are the image references used across Tekton Pipelines.
-	Images pipeline.Images
-	// DisableTaskRefResolution tells the controller not to perform
-	// resolution of task refs from the cluster or bundles.
-	DisableTaskRefResolution bool
-}
-
 // NewController instantiates a new controller.Impl from knative.dev/pkg/controller
-func NewController(namespace string, conf ControllerConfiguration) func(context.Context, configmap.Watcher) *controller.Impl {
+func NewController(fo *pipeline.Options) func(context.Context, configmap.Watcher) *controller.Impl {
```

Comment:
Ah ok after reading back over comments I think `fo` at one point stood for `FlagOptions`, but now is quite puzzling as a new reader. Suggest `opts` or `options` here just to help the reader trace the path when they're reading the function body. Similar comment for use of `fo` throughout.

Reply:
For sure, will put this in the next push.